src/main/scala/org/reactivecouchbase/rs/scaladsl/Bucket.scala

```diff
@@ -56,6 +57,8 @@ class Bucket(config: BucketConfig, onStop: () => Unit) {
     (_bucket, _bucket.async(), _bucketManager, Future.successful(_bucket.async()))
   }
 
+  private val durationToExpiry = (expiration:Duration) => ((if (expiration < 30.days) 0L else System.currentTimeMillis / 1000L) + expiration.toSeconds).toInt
```

Comment:
as scala `Duration.Inf` has a different meaning, I can create an `InfiniteExpiry` like

```
val InfiniteExpiry: Duration = Duration.Zero
```

and handle the case of `Duration.Inf` when converting duration to seconds.

But as infinite expiry is the default when writing, and you want to define the duration only if you don't want to have an infinite expiry, is it really necessary to do all that stuff ?